src/core/core/service/template_service.py

```
@@ -39,19 +40,21 @@ def create_or_update_template(template_id, base64_content):
     # Decode content
     try:
         template_content = base64.b64decode(base64_content).decode("utf-8")
-    except Exception as e:
-        return {"error": f"Failed to decode content: {e}"}, 400
+    except Exception:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)